src/ceph-volume/plugin/zfs/ceph_volume_zfs/devices/zfs/common.py

```diff
@@ -0,0 +1,95 @@
+from ceph_volume_zfs.util import arg_validators
```

Comment:
I think most (all?) of this file can be re-used from ceph-volume proper

Reply:
@alfredodeza 
Well not quite, Some things in the parser are actually used for ZFS things. 
And can be further augmented to more ZFS specific parsing

And roll_back will also need some future undoing for ZFS filesysten and things